src/custom_nodes/east_ocr/east_ocr.cpp

```diff
@@ -341,31 +415,35 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
         std::cout << "Total findings after NMS2 (non max suppression) filter: " << filteredBoxes.size() << std::endl;
     }
 
+    CustomNodeLibraryInternalManager* internalManager = static_cast<CustomNodeLibraryInternalManager*>(customNodeLibraryInternalManager);
+    std::shared_lock lock(internalManager->getInternalManagerLock());
```

Comment:
What is the reason of locking manager mutex before getting output buffers?

Reply:
In previous version of preallocation it was possible to reinitialize buffers inside initialize function(before ownership mechanism). In reinitialize there unique_lock was used for wrtiting purposes. I am not sure if we need a lock with the current implementation. If we do not need one, than model_zoo CN should also be updated.

Reply:
In the latest commit I have removed all locks including property in customNodeLibraryInternalManager